package.json

```diff
@@ -158,7 +158,7 @@
     "image-size": "0.9.3",
     "is-animated": "2.0.2",
     "isomorphic-unfetch": "3.0.0",
-    "jest": "27.0.6",
+    "jest": "29.7.0",
```

Comment:
Needed to expose Node.js globals by default. https://jestjs.io/blog/2022/04/25/jest-28#all-nodejs-globals